agent/docker/orb-agent-entry.sh

```diff
@@ -31,6 +31,14 @@ version: "1.0"
 
 visor:
   taps:
+    default_dnstap:
+      input_type: dnstap
+    default_sflow:
+      input_type: flow
```

Comment:
please add `flow_type` ... defaulting to sflow is only for backwards compatibility which we eventually want to deprecate, we want to be explicit